images/external-dns/tests/e2e.sh

```diff
@@ -1,32 +1,18 @@
 #!/usr/bin/env bash
 
-# monopod:tag:k8s
-
 set -o errexit -o nounset -o errtrace -o pipefail -x
-set +t
-
-function preflight() {
-	if [[ "${IMAGE_REGISTRY}" == "" ]]; then
-		echo "Must set IMAGE_REGISTRY environment variable. Exiting."
-		exit 1
-	fi
-
-	if [[ "${IMAGE_REPOSITORY}" == "" ]]; then
-		echo "Must set IMAGE_REPOSITORY environment variable. Exiting."
-		exit 1
-	fi
-
-	if [[ "${IMAGE_TAG}" == "" ]]; then
-		echo "Must set IMAGE_TAG environment variable. Exiting."
-		exit 1
-	fi
+
+TMPDIR=$(mktemp -d)
+
+cleanup() {
+	rm -rf $TMPDIR
 }
 
-preflight
+trap cleanup EXIT
```

Comment:
I *think* bash will only register a single trap, but that's based on fuzzy years old intuition, so don't quote me.

Reply:
since this trap is more hygiene than anything (it's cleaning up a tmpdir), I'm going to just leave this in and yolo it 🤷